Copy-edit the examples for consistency (#278)
PyO3/rust-numpy: README.md

### Write a Python module in Rust

Please see the [simple](examples/simple) example for how to get started.

> We also have examples using [ndarray-linalg](examples/linalg) and [rayon](examples/parallel).

Reviewer: I know "we" didn't originate from you, though if we're editing this anyway I think it's better to remove such pronouns (google agrees https://developers.google.com/style/pronouns#personal-pronouns).

```suggestion
There are also examples using [ndarray-linalg](examples/linalg) and [rayon](examples/parallel).
```
